finetune/util/memory.py

```diff
@@ -0,0 +1,12 @@
+import gc
+import tensorflow as tf
+
+def cleanup_sessions():
+    """
+    Clears the tensorflow session and resets the default graph.
+    Also runs gc as some of the released objects are in cycles and very large.
+    Need to be careful about using this alongside the scheduler / cache predict as it will clear the session.
+    """
+    tf.keras.backend.clear_session()
+    tf.compat.v1.reset_default_graph()
```

Comment:
So because this is used as part of finetune(), this is fine b/c during training we typically want to reset everything anyhow?  But this would problematic to call for submissions because it would wreck our model cache?